src/lucky/file_response.cr

```diff
@@ -0,0 +1,125 @@
+class Lucky::FileResponse < Lucky::Response
+  # Return a file's contents for the request.
```

Comment:
I think this comment should go above the class declaration since it doesn't correspond to any method in particular

Reply:
Ooooooo